Source/WebKit/Shared/mac/ObjCObjectGraph.mm

```diff
@@ -210,19 +200,11 @@ static bool shouldTransformGraph(id object, const ObjCObjectGraph::Transformer&
     }
 
     case ObjCType::NSArray: {
-        uint64_t size;
-        if (!decoder.decode(size))
+        std::optional<RetainPtr<NSArray>> array = decoder.decode<RetainPtr<NSArray>>();
```

Comment:
Can we use `auto` here instead?

Reply:
Right. I will update the patch to use `auto` in all cases.